system/t06_publish/update.py

```diff
@@ -217,20 +217,6 @@ class PublishUpdate5Test(BaseTest):
 
 
 class PublishUpdate6Test(BaseTest):
-    """
-    publish update: not a local repo
-    """
-    fixtureDB = True
-    fixturePool = True
-    fixtureCmds = [
-        "aptly snapshot create snap1 from mirror gnuplot-maverick",
-        "aptly publish snapshot -keyring=${files}/aptly.pub -secret-keyring=${files}/aptly.sec snap1",
-    ]
-    runCmd = "aptly publish update maverick"
-    expectedCode = 1
-
-
-class PublishUpdate7Test(BaseTest):
```

Comment:
where did test 7 go ?

Reply:
Everything is ok. Unfortunately, the following test case does not make sense any longer, because the update now also works for published snapshot repositories:
```
class PublishSwitch6Test(BaseTest):
    """
    publish switch: not a snapshot
    """
    fixtureCmds = [
        "aptly snapshot create snap1 empty",
        "aptly repo create local-repo",
        "aptly repo add local-repo ${files}",
        "aptly publish repo -keyring=${files}/aptly.pub -secret-keyring=${files}/aptly.sec -distribution=maverick local-repo",
    ]
    runCmd = "aptly publish switch maverick snap1"
    expectedCode = 1
```
I dropped that test case and renamed the subsequent cases.

Reply:
We can add additional test cases at the end for the new features.

Reply:
oh, I see, makes sense !

I would not renumber the tests, just leave a gap. some of the tests do not have numbers, but a descriptive name, which i like more somehow...